src/app.ts

```diff
@@ -45,22 +39,37 @@ import {
   NodeEnvironmentEnum
 } from "italia-ts-commons/lib/environment";
 import { CIDR } from "italia-ts-commons/lib/strings";
+import { UrlFromString } from "italia-ts-commons/lib/url";
 import { ServerInfo } from "../generated/public/ServerInfo";
 
 import AuthenticationController from "./controllers/authenticationController";
+import MessagesController from "./controllers/messagesController";
+import NotificationController from "./controllers/notificationController";
 import PagoPAController from "./controllers/pagoPAController";
 import PagoPAProxyController from "./controllers/pagoPAProxyController";
+import ProfileController from "./controllers/profileController";
+import ServicesController from "./controllers/servicesController";
 import SessionController from "./controllers/sessionController";
+import UserMetadataController from "./controllers/userMetadataController";
 
 import { log } from "./utils/logger";
 import checkIP from "./utils/middleware/checkIP";
 
 import getErrorCodeFromResponse from "./utils/getErrorCodeFromResponse";
 
+import NotificationService from "./services/notificationService";
 import { User } from "./types/user";
 import { toExpressHandler } from "./utils/express";
 
+import MessagesService from "./services/messagesService";
+import PagoPAProxyService from "./services/pagoPAProxyService";
+import ProfileService from "./services/profileService";
+import RedisSessionStorage from "./services/redisSessionStorage";
+import RedisUserMetadataStorage from "./services/redisUserMetadataStorage";
+import TokenService from "./services/tokenService";
+
 const defaultModule = {
+  SPID_STRATEGY: generateSpidStrategy(),
```

Comment:
why all caps? as far as I understand this should be probably named `currentSpidStrategy` ?

Reply:
fixed in 5650e7f